### ISSUE TITLE
[Add] : Hand Gesture Recognition Model

### ISSUE THREAD

**Issue by AKACHI-4** [OpenCodeEra/ML-X#11]
## What ?

I would like to add a model for recognizing hand gesture-based outputs.

- ### Algorithm

- ### Dataset

> Discuss your algorithm and dataset down below..

## Code of Conduct

- [x] I agree to follow this project's Code of Conduct

<br>
<br>

> **Kindly assign it to me under _HacktoberFest'23_** 
> **I will surely complete it ASAP**

**Comment by VishnuAmit** [comment]
Hey! I am interested in contributing to this model. Can I have some more details of it so that I can start working right away !

**Comment by AKACHI-4** [comment]
> Hey! I am interested in contributing to this model. Can I have some more details of it so that I can start working right away !

Hey @VishnuAmit, yah, sure... could you plz elaborate ur doubt so that I can provide specific answer to u.

**Comment by VishnuAmit** [comment]
Any specific algorithms or datasets to keep in mind when i work?? Or its just in general?

**Comment by AKACHI-4** [comment]
> Any specific algorithms or datasets to keep in mind when i work?? Or its just in general?

No, we are not specific to any one dataset or algorithm. You have the freedom to choose any related algorithm and a well-suited dataset, 
but keep in mind that the dataset you are using should have a sufficient number of parameters.

**Comment by VishnuAmit** [comment]
Alright. So Can you assign me for this contribution under hacktoberfest'23 or should I send you the PR first with the model ?

**Comment by AKACHI-4** [comment]
> Alright. So Can you assign me for this contribution under hacktoberfest'23 or should I send you the PR first with the model ?

Yah.. I will. But before that, could you please provide a dataset or algorithm that you would like to use for the task.. ?

**Comment by VishnuAmit** [comment]
I am gonna be using a public dataset available from kaggle and CNN algorithm.
Dataset link : https://www.kaggle.com/datasets/gti-upm/leapgestrecog

**Comment by AKACHI-4** [comment]
@VishnuAmit Go Ahead... Also plz remember to add notes in your `notebook.ipynb` so others can understand it as well.

**Comment by AKACHI-4** [comment]
Yah okay .. Happy for your excitement...
